src/API/Trace/SpanBuilderInterface.php

```diff
@@ -15,14 +15,14 @@
 interface SpanBuilderInterface
 {
     /**
-     * Sets the parent {@see Context} to use.
+     * Sets the parent {@see ContextInterface} to use.
      *
      * If no {@see SpanInterface} is available in the provided context, the resulting span will become a root span,
      * as if {@see SpanBuilderInterface::setNoParent} was called.
      *
      * Defaults to {@see Context::getCurrent} when {@see SpanBuilderInterface::startSpan} was called if not explicitly set.
      */
-    public function setParent(Context $parentContext): SpanBuilderInterface;
+    public function setParent(ContextInterface $parentContext): SpanBuilderInterface;
```

Comment:
```suggestion
    public function setParent(ContextInterface|false|null $parentContext): SpanBuilderInterface;
```
Should this be changed (same typehint is already used in metrics API/SDK for context parameters) and `::setNoParent()` be removed?

Reply:
Yes that makes sense. Are false and null equivalent, here?